src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Конструктор резюме</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/main.css">
+</head>
+<body>
+    <section class="blocking">
+        <div class="main">
+            <h1 class="h1 center">Конструктор резюме</h1>
+            <p class="text">Этот конструктор резюме разработан для того, чтобы помочь вам продемонстрировать свои навыки, опыт и квалификацию в понятной, современной и организованной форме. Этот шаблон идеально подходит для людей из различных отраслей, которые стремятся произвести сильное первое впечатление.</p>
+            <form class="center" action="cv_1.html">
+                <input class="btn_create h2" type="submit" value="Создать резюме">
+            </form>
+        </div>
+    </section>
+    <section class="blocking pics">
+        <div class="pics_1">
+            <img class="imgS" src="./img/img1.jpg" alt="Картинка 1">
```

Comment:
Для скринридера важно корректное содержимое `alt`. Можно оставить `alt` пустым в некоторых случаях. 
```suggestion
            <img class="imgS" src="./img/img1.jpg" alt="Подробное описание картинки 1">
```
См. подробнее https://doka.guide/html/alt/